tests_peerscout_api/end2end_tests/end2end_test.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import requests
+
+LOGGER = logging.getLogger(__name__)
+
+request_json = """{
```

Comment:
maybe we could just load `example-data/peerscout-api-request1.json`?

Reply:
it is added